### DISCUSSION TITLE
Custom CSS: Maximum text width no longer works in Zettlr 3

### DISCUSSION THREAD

**Discussion by jelkejelke** [Zettlr/Zettlr#4609] [Q&A]
I really appreciated that I could set maximum text width using CSS in the Zettlr 2.3. However, it seems the snippet provided in the documentation no longer works in Zettlr 3. I've tried to figure out how to updated the snippets, but have been unsuccessful so far. can anybody help me out with this?

To be sure, I'm referring to the following custom CSS, as provided in the documentation:
```
#editor {
  --side-margin: calc( 50vw - 30em ); 
}

#editor .CodeMirror {
  margin-left: var(--side-margin);
}

#editor .CodeMirror-scroll {
  padding-right: var(--side-margin);
}
```

Edit: I've been able to achieve something that resembles the above by editing the CSS using the developer tools. Not entirely sure how to explain where I made the edits, but perhaps the CSS selector is of help: 
`#window-content > div > div.view.view-border > div > div:nth-child(1) > div > div.editor-container > div > div > div.cm-scroller.cm-emacsMode` 
I've been able to adjust the padding (set to +- 500px left and right)  here to achieve the desired behaviour/look, but haven't been able to translate this in appropriate custom css and make it responsive to window resizing.

**Comment by ProodjePindakaas** [comment]
I've found the same, that the old code indeed does not work anymore.
So I'm not very proficient in CSS, but I've managed to get it responsive to the window size.
Here is the custom css I used: 
```
.main-editor-wrapper .cm-content {
  max-width: 40em;  /* change value to your preferred width */
  margin-right: auto;
}

.main-editor-wrapper .cm-gutters {
  margin-left: auto;
}
```
